source/draw/gpu/opengl/_RecycleBin.ooc

```diff
@@ -23,7 +23,7 @@ import OpenGLPacked, OpenGLMonochrome, OpenGLBgra, OpenGLBgr, OpenGLUv
 import threading/Mutex
 
 version(!gpuOff) {
-RecycleBin: class {
+_RecycleBin: class {
```

Comment:
This will eventually be replaced by the generic `RecycleBin` class so I temporarily renamed it to a private class.